packages/appstore/src/components/app-card/app-card.tsx

```diff
@@ -0,0 +1,316 @@
+                                {balance} {currency_name}
+                            </p>
+                        </div>
+                    </div>
+                )}
+            </>
+        </div>
+    );
+};
+
+const UnlinkedAppcard = ({ app_name, app_icon, size }: TUnlinkedAppCard) => {
+    const size_el = size && `appstore-app-card__block--unlinked__wrapper--${size}`;
+
+    return (
+        <div className={classNames('appstore-app-card__block--unlinked', size_el)}>
+            <>
+                {size === 'small' ? (
+                    <div className='appstore-app-card__small-block-wrapper--unlinked'>
+                        <>
+                            {app_icon ? (
+                                <Icon icon={app_icon} width='24' height='24' />
+                            ) : (
+                                <div className='appstore-app-card--appcard-placeholder'>&nbsp;</div>
+                            )}
+                        </>
+                    </div>
+                ) : (
+                    <div className='appstore-app-card__block--unlinked__wrapper'>
+                        <div className='appstore-app-card__block--unlinked__wrapper--icon'>
+                            <>
+                                {app_icon ? (
+                                    <Icon icon={app_icon} width='24' height='24' />
+                                ) : (
+                                    <div className='appstore-app-card--appcard-placeholder'>&nbsp;</div>
+                                )}
+                            </>
+                        </div>
+                        <p className='appstore-app-card__block--unlinked__wrapper--text'>{app_name}</p>
+                    </div>
+                )}
+            </>
+        </div>
+    );
+};
+
+const AppCard = ({ app_card_details, size, dark, linked, checked }: TCFDAppCard) => {
+    return (
+        <>
+            {linked ? (
+                <LinkedAppCard
+                    app_name={app_card_details?.app_name}
+                    app_icon={app_card_details?.app_icon}
+                    wallet_name={app_card_details?.wallet_name}
+                    wallet_icon={app_card_details?.wallet_icon}
+                    currency_name={app_card_details?.currency_name}
+                    balance={app_card_details?.balance}
+                    size={size}
+                    dark={dark}
+                    checked={checked}
+                />
+            ) : (
+                <UnlinkedAppcard
+                    size={size}
+                    app_name={app_card_details?.app_name}
+                    app_icon={app_card_details?.app_icon}
+                />
+            )}
+        </>
+    );
+};
+
+const AppstoreAppCard = ({
+    account_type,
+    app_card_details,
+    size,
+    checked,
+    dark,
+    linked,
+    faded,
+    disabled,
+}: TCFDAppCard) => {
+    const [is_hovered, set_hovered] = React.useState(false);
```

Comment:
```suggestion
    const [is_hovered, setHovered] = React.useState(false);
```
Function name should be in camel case - https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#naming-conventions-functions